docs/CHANGELOG.md

```diff
@@ -42,6 +43,7 @@ Starting in `v2.9.0`:
 ### Removed
 
 - Removed the ability to set a next nonce value for an address through CLI
+- Removed `Optimism2` as a supported gas estimator mode
```

Comment:
Is this important to do? Because it may require a two release delay, after warning node ops about the change.

Reply:
IIRC we migrated from `Optimism2` to `optimismBedrock` during Bedrock, so there shouldn't be any user using that config. Also, the original task included contacting NOPs about the new name of the L2Suggested estimator in the first place so this seemed like a good opportunity to kill two birds with one stone.

Reply:
cc @chudilka1